devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -184,6 +192,10 @@ def pause() {
 	zigbee.command(CLUSTER_WINDOW_COVERING, COMMAND_PAUSE)
 }
 
+def presetPosition() {
+    setLevel(preset ?: 50)
```

Comment:
it looks like zwave window shade uses `setLevel(preset ?: state.preset ?: 50)` idk which is more correct

Reply:
@greens Yeah, I saw that. I'm not sure that I completely follow the logic. I'm not sure if preferences are accessible through state like that? They could be. I've always seen just directly using the name as a variable, or settings.prefName. I'll check this out as I am testing it.